ci/test/00_setup_env_native_multiprocess.sh

```diff
@@ -11,7 +11,8 @@ export DOCKER_NAME_TAG=ubuntu:20.04
 export PACKAGES="cmake python3 python3-pip llvm clang"
 export DEP_OPTS="DEBUG=1 MULTIPROCESS=1"
 export GOAL="install"
-export BITCOIN_CONFIG="--enable-debug CC=clang CXX=clang++"  # Use clang to avoid OOM
+# The multiprocess feature is currently incompatible with the experimental syscall sandbox feature (-sandbox=<mode>).
```

Comment:
I've now added  an`--nosandbox` test runner option, and `export TEST_RUNNER_EXTRA="--nosandbox"` for this job.

What would be the best way to detect usage of the multiprocess feature at runtime in `AppInitParameterInteraction`?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/20487#discussion_r680406422

> What would be the best way to detect usage of the multiprocess feature at runtime in `AppInitParameterInteraction`?

You could add a `bool ipc_enabled` parameter to `AppInitParameterInteraction` and pass `node.init->ipc()` as the argument value where it's called in `AppInit`, and `false` as the argument value where it's called by the test framework.

Handling this in `AppInitParameterInteraction` shouldn't strictly be necessary to make tests pass, but it could be used to print a more helpful error message if there's an attempt to use `bitcoin-node` IPC functionality when sandboxing is enabled. The `echoipc` RPC method is the only IPC functionality that's merged so far, so it would be nice to make this change, but it could also be done later with #10102.